request-authentication.md

```diff
@@ -6,13 +6,85 @@ pagination_next: null
 pagination_prev: null
 ---
 
+import ApiSchema from '@theme/ApiSchema';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
 # Request authentication
 
 A notification request from webhooks can be verified by using the `host`, `x-ms-date`, `x-ms-content-sha256`, and `authorization` headers with the
 unique secret received when the webhook was registered. The authorization is a
 sha256 HMAC hash of the request date, content, and URI that is created using the
 webhook secret.
 
+<Tabs
+    defaultValue="js" 
+    groupId="example-choice" 
+    values={[ {label: 'JavaScript', value: 'js'}, {label: '.Net C#', value: 'csharp'}, ]}>
+
+<TabItem value="js">
+
+To run: `node webhook.mjs`
```

Comment:
Where's webhook.mjs, and is it okay to require a node installation?
Do we have a preference wrt. programming language in the examples @cloveras?

Reply:
Removed in [e422710](https://github.com/vippsas/webhooks-api/pull/23/commits/e422710fb6dfc82aa8cbcbc2cb2ef26d4106aaf1)